Makefile

```diff
@@ -4,8 +4,8 @@ all : build
 
 build : clean
 	mkdir -p $(BIN_DIR) $(BIN_DIR)/amd64-linux
-	go build -o $(BIN_DIR) github.com/eagraf/habitat/cmd/habitat github.com/eagraf/habitat/cmd/habitatctl
-	GOOS=linux GOARCH=amd64 go build -o $(BIN_DIR)/amd64-linux github.com/eagraf/habitat/cmd/habitat github.com/eagraf/habitat/cmd/habitatctl
+	go build -o $(BIN_DIR) github.com/eagraf/habitat/cmd/habitat github.com/eagraf/habitat/pkg/habitatctl
```

Comment:
Could we just move the `client` library into `pkg`, and keep the rest of `habitatctl` where it is? We could also name it something like `habitat_client`